public/styles/styles.scss

```diff
@@ -11,3 +13,8 @@ body {
   background: globalVariables.$color-6;
   color: globalVariables.$color-0;
 }
+
+h1, h2, h3, h4, h5, h6, p {
```

Comment:
Not bad to apply for the html itself, refer to this [link](https://meyerweb.com/eric/tools/css/reset/) easier than tracking particular items.